crowdloans/rewards/polkadot.json

```diff
@@ -1402,7 +1402,8 @@
     },
     "parallelBonus": {
       "items": [
-        { "formula": "0.10", "logo": "/static/media/para.e1be6567.svg" }
+        { "formula": "0.05", "logo": "/static/media/para.e1be6567.svg" },
```

Comment:
to confirm, is this parallel formula apply to all the projects? all the parallel referal bonus calcualtion should be *0.05 on front end

Reply:
this is for 2021 (efi)